tests/unit-tests/test-class-sensei.php

```diff
@@ -155,10 +155,12 @@ private function resetUpdateOptions() {
 	 *
 	 * The version number in the plugin information block should match the version number specified in the code.
 	 */
-	function testVersionNumber() {
+	public function testVersionNumberIsCorrect() {
+		/* Arrange. */
+		$plugin_date = get_plugin_data( SENSEI_LMS_PLUGIN_FILE );
```

Comment:
`$plugin_data`? :)

Reply:
Oh, thanks! Fixed in e7a7270e50945f66c7645979bc8b751ac7e21a17.